src/schema/annotation.yaml

```diff
@@ -170,6 +170,9 @@ classes:
       A set of genes or gene products in which all members are orthologous
     id_prefixes:
       - KEGG.ORTHOLOGY  ## KO number
+        slot_usage:
```

Comment:
yaml syntax err. you can't mix lists and dicts. I think you intend this block to be at the level of the class

Reply:
Thanks. Yeah, that's what I had in my notes but I didn't see how that could bind the pattern to the appropriate `id_prefixes` without lexically parsing 'KEGG.ORTHOLOGY' out of the pattern.

Reply:
OK, now this

```yaml
  orthology group:
    is_a: functional annotation term
    description: >-
      A set of genes or gene products in which all members are orthologous
    slot_usage:
        id:Abstract
            pattern: "^(KEGG.ORTHOLOGY:K\\d{5}$"
```

is giving me

> yaml.scanner.ScannerError: mapping values are not allowed here

Reply:
Moved the `id` slot usage to `functional annotation` slot usage and`test/data/functional_annotation_set_valid.json` passes `make test-jsonschema` 

The `raw.functional_annotation` collection in Polyneme's MongoDB has `FunctionalAnnotation` objects taking string objects to the `has function` property, not `FunctionalAnnotationTerm` objects as advertised by the [web-based schema documentation.](https://microbiomedata.github.io/nmdc-schema/FunctionalAnnotation.html) 

@wdduncan and @cmungall already drew my attention to the fact that `FunctionalAnnotationTerm` isn't completely implemented yet.